packages/lab/src/date-picker/DatePickerRangePanel.tsx

```diff
@@ -119,13 +117,9 @@ export interface DatePickerRangePanelProps<TDate extends DateFrameworkType>
     >
   >;
   /**
-   * Props to be passed to the start date CalendarWeekHeader component.
+   * Props to be passed to the start date CalendarGrid component.
    */
-  StartCalendarWeekHeaderProps?: CalendarWeekHeaderProps;
```

Comment:
Start / end props removal is not mentioned in changeset?

Reply:
good spot, updated changeset